api/src/main/java/org/openmrs/OrderType.java

```diff
@@ -25,10 +25,10 @@
 public class OrderType extends BaseChangeableOpenmrsMetadata {
 	
 	public static final long serialVersionUID = 23232L;
-	
-	public static final String DRUG_ORDER_TYPE_UUID = "131168f4-15f5-102d-96e4-000c29c2a5d7";
-	
-	public static final String TEST_ORDER_TYPE_UUID = "52a447d3-a64a-11e3-9aeb-50e549534c5e";
+//
```

Comment:
Thanks @tendomart , i think this should be removed since you called them above

Reply:
Thanks @sherrif10 , i was actually trying to reserve the uuid feature until the className feature has been approved so we can either completely remove it the **unique uuid fields** are confirmed to have no other dependency .Do you get the idea